showcase/app/templates/components/advanced-table.hbs

```diff
@@ -174,7 +174,7 @@
         (hash key="email" label="Email")
         (hash key="role" label="Role")
       }}
-      @hasStickyHeader={{true}}
+      @maxHeight="400px"
```

Comment:
[suggestion] here we're testing the condition where only the `@maxHeight` is declared, which implicitly sets the `@hasStickyHeader` to `true`; maybe we should add also a use case where  `@maxHeight` is declared, but `@hasStickyHeader` is explicitly set to `false`?

Reply:
True! I have a test for this but not a showcase example. Will update.

Reply:
I ended up deleting the `shw-component-advanced-table-fixed-dimensions-wrapper` class and using the `shw-component-advanced-table-fixed-width-wrapper` class instead in combination with the `maxHeight` + `hasStickyHeader={{false}}` a few times in the showcase. It should be good now.